<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Help Weath</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(65, 65, 65)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 20px;
color: white;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 25px;
font-weight: 900;
color: white;</string>
    </property>
    <property name="text">
     <string>Введите регион</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 13px;</string>
    </property>
    <property name="text">
     <string>Введите в поле выше регион информация о котором вам нужна</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton {
	background-color: rgb(125, 125, 125);
	color: white;
	font-size: 20px;
	border-radius: 3px;
	height: 24px;
}

QPushButton#pushButton:hover {
	background-color: rgb(157, 157, 157);
	border-radius: 7px;
}

QPushButton#pushButton:focus {
	background-color: rgb(77, 77, 77);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>Посмотреть</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 19px;
color: white;
</string>
    </property>
    <property name="text">
     <string>Температура</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 19px;
color: white;
</string>
    </property>
    <property name="text">
     <string>Скорость ветра</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>490</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 19px;
color: white;
</string>
    </property>
    <property name="text">
     <string>Влажность</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_2 {
	background-color: rgb(125, 125, 125);
	color: white;
	font-size: 20px;
	border-radius: 3px;
	height: 24px;
}

QPushButton#pushButton_2:hover {
	background-color: rgb(157, 157, 157);
	border-radius: 7px;
}

QPushButton#pushButton_2:focus {
	background-color: rgb(77, 77, 77);
	border-radius: 5px;
}</string>
    </property>
    <property name="text">
     <string>В избранное</string>
    </property>
   </widget>
   <widget class="QPushButton" name="helpBut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#helpBut {
	color: white;
	font-size: 15px;
	border-radius: 3px;
	height: 24px;
}

QPushButton#helpBut:hover {
	background-color: rgb(157, 157, 157);
	border-radius: 7px;
}

</string>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="temper_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
color: white;
</string>
    </property>
    <property name="text">
     <string>°C</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="speed_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
color: white;
</string>
    </property>
    <property name="text">
     <string>м/с</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="vlazhnost_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>480</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 30px;
color: white;
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
